Examples/Io/Root/src/RootTrajectoryWriter.cpp

```diff
@@ -265,22 +266,34 @@ FW::ProcessCode FW::RootTrajectoryWriter::writeT(
   // Loop over the trajectories
   int iTraj = 0;
   for (const auto& traj : trajectories) {
-    /// Collect the information
     m_trajNr = iTraj;
+    if (not traj.hasTrajectory()) {
+      continue;
```

Comment:
That's not a warning case? I'm not sure I quite understand the case where there is an entry in `trajectories` here, but `.hasTrajectory()` is false? Is that if no track at all is found for a truth particle?

Reply:
It's because there could an empty track. For example, in the fitting algorithm, an empty track is still created and pushed to the collection if the there is an empty proto track or the fitting fails (If I remember correctly, this is added by @msmk0 for some consideration). If we agree that we don't store an empty track, then there is indeeed no need to do the `hasTrajectory()` here.